pkg/metrics/stats_reporter.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
+	sdkmetric "go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregation"
+)
+
+var (
+	// defines the metrics that are collected
+	verificationDuration instrument.Int64Histogram
+	mutationDuration     instrument.Int64Histogram
+	verifierDuration     instrument.Int64Histogram
+	systemErrorCount     instrument.Int64Counter
+)
+
+const (
+	scope = "github.com/deislabs/ratify"
+
+	// metric names
+	metricNameVerificationDuration = "ratify_verification_request"
+	metricNameMutationDuration     = "ratify_mutation_request"
+	metricNameVerifierDuration     = "ratify_verifier_duration"
+	metricNameSystemErrorCount     = "ratify_system_error_count"
+)
+
+// NewStatsReporter creates a new StatsReporter
+func initStatsReporter() error {
+	var err error
+	views := []sdkmetric.View{
+		sdkmetric.NewView(
+			sdkmetric.Instrument{
+				Name:  metricNameVerificationDuration,
+				Scope: instrumentation.Scope{Name: scope},
+			},
+			sdkmetric.Stream{
+				Aggregation: aggregation.ExplicitBucketHistogram{
+					Boundaries: []float64{0, 100, 400, 700, 1000, 1300, 1600, 1900},
```

Comment:
suggestion to add some comments to these view so configuration is more readable.  For example, Why do we use 100,400, 700 ?